### PR TITLE
Labs/3

### PR DESCRIPTION
**Pull request by artegful**

### CODE REVIEW

**Comment by artegful** [comment]
https://artegful.github.io/itirod

**Comment by nastik1811** [comment]
What's this?
image

File: style.css [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+  align-items: center;
+  flex-direction: column;
+  clear: right;
+}
+
+p.timer {
+  margin: 0;
+  font-size: xx-large;
+  color: blue;
+  float: right;
```
Comment (nastik1811) [inline style.css]: don't use floats

File: style.css [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+.unselectable {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+@media (max-aspect-ratio: 1/1) {
```
Comment (nastik1811) [inline style.css:207]: How does it work?

File: settings.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            <div class="settings-button-container">
+              <div class="settings-button blue-theme unselectable">
+                blue
+              </div>
+              <div class="settings-button red-theme unselectable">
+                red
+              </div>
+              <div class="settings-button funk-theme unselectable">
+                funk
+              </div>
```
Comment (nastik1811) [inline settings.html]: they should be buttons

**Review by nastik1811 — CHANGES_REQUESTED** [review]
At this point, you should also have styles for all the color themes that will be in your application

**Comment by artegful** [comment]
> What's this? image

Removed

File: style.css (hunk 2 above)
Comment (artegful) [reply style.css:207]: Applies to screens that are wider than squares

File: settings.html (hunk 3 above)
Comment (artegful) [reply settings.html]: Fixed

File: style.css (hunk 1 above)
Comment (artegful) [reply style.css]: Removed

**Review by nastik1811 — APPROVED** [review]
ok, `7`

**Pull request merged**